src/antsibull_docs/cli/antsibull_docs.py

```diff
@@ -41,34 +52,33 @@
     is_wildcard_collection_name,
 )
 from ..schemas.app_context import DocsAppContext  # noqa: E402
-from .doc_commands import (  # noqa: E402
-    collection,
-    collection_plugins,
-    current,
-    devel,
-    lint_docs,
-    plugin,
-    sphinx_init,
-    stable,
-)
 
 # pylint: enable=wrong-import-position
 
 
 mlog = log.fields(mod=__name__)
 
+
+def _create_loader(module: str, function: str) -> Callable[[], Callable[[], int]]:
+    def load():
+        module_obj = import_module(f"antsibull_docs.cli.doc_commands.{module}")
+        return getattr(module_obj, function)
```

Comment:
Does running `import_module` every time have much of a performance overhead?

Reply:
Line 64 is ran at most once (in total) per antsibull-docs invocation, so I don't think so.